src/libtrx/config/file.c

```diff
@@ -165,6 +193,9 @@ bool ConfigFile_Write(const char *path, void (*dump)(JSON_OBJECT *root_obj))
     if (old_data != NULL) {
         Memory_FreePointer(&old_data);
     }
+    if (enforced_data != NULL) {
+        Memory_FreePointer(&enforced_data);
+    }
```

Comment:
Memory_Free family already checks for nulls so no need for this double safety.

Reply:
Thanks, updated.